main.go

```diff
@@ -17,18 +17,21 @@ func main() {
 	}
 	conf.SetVersion(version)
 
-	u, err := cmd.NewSelfUpdate()
-	if err != nil {
-		// not fatal
-		cmd.GetRootCmd().Printf("error checking for updates: %v\n", err)
-	}
-	if u != nil {
-		m, err := u.Run()
+	// disable update check if running a locally built version
+	if version != "0.0.0-dev" {
+		u, err := cmd.NewSelfUpdate()
 		if err != nil {
+			// not fatal
 			cmd.GetRootCmd().Printf("error checking for updates: %v\n", err)
```

Comment:
can it return after this?

Reply:
no the execute is what initializes the tool.